bears/python/VultureBear.py

```diff
@@ -1,45 +1,45 @@
-import re
-from shutil import which
-
 from coalib.bears.GlobalBear import GlobalBear
-from dependency_management.requirements.PipRequirement import PipRequirement
-from coalib.misc.Shell import run_shell_command
 from coalib.results.Result import Result
+from dependency_management.requirements.PipRequirement import PipRequirement
+from vulture import Vulture
+
+
+def _find_unused_code(filenames):
+    """
+    :param filenames: List of filenames to check.
+    :return: Generator of Result objects.
+    """
+
+    def file_lineno(item):
+        return (item.filename.lower(), item.lineno)
+
+    vulture = Vulture()
+    vulture.scavenge(filenames)
+    for item in sorted(
+            vulture.unused_funcs + vulture.unused_imports +
+            vulture.unused_props + vulture.unused_vars +
+            vulture.unused_attrs, key=file_lineno):
+        message = 'Unused {0}: {1}'.format(item.typ, item)
+        yield Result.from_values(origin='VultureBear',
+                                 message=message,
+                                 file=item.filename,
+                                 line=item.lineno)
 
 
 class VultureBear(GlobalBear):
     LANGUAGES = {'Python', 'Python 3'}
-    REQUIREMENTS = {PipRequirement('vulture', '0.10.0')}
+    REQUIREMENTS = {PipRequirement('vulture', '0.14.0')}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     ASCIINEMA_URL = 'https://asciinema.org/a/82256'
     CAN_DETECT = {'Unused Code'}
 
-    EXECUTABLE = 'vulture'
-    OUTPUT_REGEX = re.compile(
-        r'(?P<filename>.*):(?P<line>.*):\s*(?P<message>.*)')
-
-    @classmethod
-    def check_prerequisites(cls):
-        return ('Vulture is missing. Make sure to install it using '
-                '`pip3 install vulture`.'
-                if which('vulture') is None else True)
-
     def run(self):
         """
         Check Python code for unused variables and functions using `vulture`.
 
-        See <https://bitbucket.org/jendrikseipp/vulture> for more information.
+        See <https://github.com/jendrikseipp/vulture> for more information.
```

Comment:
This is an unrelated change, and should be a separate commit, or even a separate PR, and as you will be modifying this line anyway, you can move it to a `SEE_MORE` attribute (c.f. https://github.com/coala/coala-bears/issues/1136)